.gitignore

```diff
@@ -1,3 +1,6 @@
+# VS Code
+.vscode/
+
```

Comment:
Yes. Sorry, should have put it in another commit. .vscode is a temp directory from visual studio code.